api/routes.py

```diff
@@ -72,7 +72,7 @@ def getGeneralMap(kind: str):
         else:
             return abort(404)
 
-    @api.route("/bike_count/stations")
+    @api.route("map/bike_count/historic_stations")
```

Comment:
Isn't there an `/` missing at the beginning of `map/bike_count/historic_stations` ?

Reply:
Yes there is, thanks for spotting!